mousecursor.go

```diff
@@ -0,0 +1,61 @@
+package flutter
+
+import (
+	"errors"
+	"fmt"
+	"github.com/go-flutter-desktop/go-flutter/plugin"
+	"github.com/go-gl/glfw/v3.3/glfw"
+)
+
+const mousecursorChannelName = "flutter/mousecursor"
+
+// mousecursorPlugin implements flutter.Plugin and handles method calls to the
+// flutter/mousecursor channel.
+type mousecursorPlugin struct {
+	channel    *plugin.MethodChannel
+	window     *glfw.Window
+	lastCursor *glfw.Cursor
+}
+
+var defaultMousecursorPlugin = &mousecursorPlugin{}
+
+func (p *mousecursorPlugin) InitPlugin(messenger plugin.BinaryMessenger) error {
+	p.channel = plugin.NewMethodChannel(messenger, mousecursorChannelName, plugin.StandardMethodCodec{})
+	return nil
+}
+func (p *mousecursorPlugin) InitPluginGLFW(window *glfw.Window) error {
+	p.window = window
+	p.channel.HandleFuncSync("activateSystemCursor", p.handleActivateSystemCursor)
```

Comment:
The HandleFuncSync func can be moved in to InitPlugin func. Then p.channel can be removed